config/environments/development.rb

```diff
@@ -131,8 +126,9 @@
 
   ###################################################################
   # Location of the public and private key for the git user on the system
-  GITOLITE_SETTINGS = { public_key: "/home/git/git.pub",
-                        private_key: "/home/git/.ssh/id_rsa" }
+  GITOLITE_SETTINGS = { public_key: "/home/vagrant/git.pub",
+                        private_key: "/home/vagrant/.ssh/id_rsa",
+			host: "localhost" }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Tab detected.